bin/anthology/utils.py

```diff
@@ -437,15 +437,11 @@ def parse_element(xml_element):
             value = {"filename": element.text, "src": element.get("src")}
         elif tag == "video":
             # Treat videos the same way as other attachments
-            tag = "attachment"
+            tag = "video"
             # Skip videos where permission was not granted (these should be marked as private anyway in Vimeo)
             if element.get("permission", "true") == "false":
                 continue
-            value = {
-                "filename": element.get("href"),
-                "type": element.get("tag", "video"),
-                "url": element.get("href"),
-            }
+            value = element.get("href")
```

Comment:
I'm guessing we don't use these other values here?

Reply:
These are the expected values for type "attachment", but "video" is now separate. I wonder if we might have a use for additional fields in the future, though, but it probably wouldn't be too difficult to refactor again later.

Reply:
@mjpost @mbollmann Yes, it makes more sense to refactor in the future (with little effort).